addons/sourcemod/scripting/superzombiefortress.sp

```diff
@@ -184,6 +184,7 @@ enum struct ClientClasses
 	float flHorde;
 	float flMaxSpree;
 	float flMaxHorde;
+	int iMoraleValue;
```

Comment:
May well then go for `float flMoraleValue` when calculations is done in float anyway